packages/fake-driver/lib/fake-app.js

```diff
@@ -1,13 +1,11 @@
-import B from 'bluebird';
-import fs from 'fs';
+import {readFile} from 'fs/promises';
```

Comment:
i don't know if we can have an eslint rule about this, but we differentiate named imports from object destructuring in our whitespace:

```
import { foo } from 'bar';
export { thing1, thing2 };
```
vs
```
const {foo} = bar;
const thing = {x: 1, y: 2};
```

Reply:
eslint is obviously not configured to care about this...  I do pretty well with an eslint + prettier combo and would be gung-ho about adding prettier here, if you are interested.

Reply:
I'm up for that. Are you saying it could help us make this distinction?

Reply:
I tried it, actually, and prettier makes no distinction.  you may have a tough time making this work.